sdk/eventhub/eventhubs-checkpointstore-blob/package.json

```diff
@@ -121,6 +121,15 @@
     "typedoc": "0.15.2"
   },
   "//sampleConfiguration": {
-    "skipFolder": true
+    "skipFolder": true,
+    "productName": "Azure Event Hubs",
```

Comment:
Since this is used to generate the page title, it probably needs something more specific, like `Azure Event Hubs - Checkpointstore`, or whatever you think would be appropriate.

Currently, this causes the title to display as "Azure Event Hubs samples for JavaScript", same as the main EH package.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/15137/commits/a8b1a85e6b4de25838e0797a436ed9fdaa29d0d2